QtCollider/primitives/prim_misc.cpp

```diff
@@ -168,7 +168,7 @@ QC_LANG_PRIMITIVE( Qt_SetGlobalPalette, 1, PyrSlot *r, PyrSlot *a, VMGlobals *g
 {
   if( !QcApplication::compareThread() ) return QtCollider::wrongThreadError();
 
-  QPalette p = QtCollider::get( a );
+  QPalette p = (QPalette&&) QtCollider::get(a);
```

Comment:
I'm not certain about this code. VS was complaining that there were two constructors it could choose to create p, one took a QPalette&&, the other took some kind of button.. It seemed the safest choice.